src/style.css

```diff
@@ -1,3 +1,113 @@
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+
 }
+
+.header {
+  background: #fff;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  position: relative;
+}
+
+@media (max-width: 1024px) {
+  .header {
+    width: 1024px;
+  }
+
+}
+@media (min-width: 1200px) {
+  .header {
+    width: 100%;
+  }
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-left: 50px;
+}
+
+.logo-moyo {
+  display: flex;
+}
+
+.nav {
+  position: absolute;
+  right: 50px;
+}
```

Comment:
Let's try a different way, just use the `flexbox` properties for the `.header` to move the blocks apart

Reply:
Done